models/src_hubspot.yml

```diff
@@ -188,6 +188,8 @@ sources:
       - name: contact_form_submission
         identifier: "{{ var('hubspot_contact_form_submission_identifier', 'contact_form_submission')}}"
         description: Table containing contact form submission information
+        config:
+          enabled: "{{ var('hubspot_marketing_enabled', true) and var('hubspot_contact_form_submission_enabled', true) }}"
```

Comment:
Did we already have this table included in the source file but not in a staging model?